p2p/protocol/holepunch/coordination.go

```diff
@@ -0,0 +1,302 @@
+package holepunch
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	logging "github.com/ipfs/go-log"
+	"github.com/jpillora/backoff"
+	"github.com/libp2p/go-libp2p-core/host"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+	holepunch_pb "github.com/libp2p/go-libp2p/p2p/protocol/holepunch/pb"
+	"github.com/libp2p/go-libp2p/p2p/protocol/identify"
+	"github.com/libp2p/go-msgio/protoio"
+	ma "github.com/multiformats/go-multiaddr"
+	manet "github.com/multiformats/go-multiaddr/net"
+)
+
+// TODO Should we have options for these ?
+const (
+	protocol         = "/libp2p/holepunch/1.0.0"
+	maxMsgSize       = 4 * 1024 // 4K
+	holePunchTimeout = 2 * time.Minute
+	dialTimeout      = 60 * time.Second
+	maxRetries       = 4
+)
+
+var (
+	log = logging.Logger("p2p/holepunch")
```

Comment:
use a dash, this is inconsistent with the other logger names we use across the stack.

Reply:
done.